components/logins/src/engine.rs

```diff
@@ -44,8 +44,8 @@ impl PasswordEngine {
         self.db.get_by_id(id)
     }
 
-    pub fn get_by_hostname(&self, hostname: &str) -> Result<Vec<Login>> {
-        self.db.get_by_hostname(hostname)
+    pub fn get_by_base_domain(&self, base_domain: &str) -> Result<Vec<Login>> {
```

Comment:
I guess technically the "hostname" terminology was correct… I wasn't thinking about IP addresses when I proposed the "…BaseDomain" name… I guess we could leave the old name but then document that subdomains are returned? Is there not an equivalent of JavaDoc for the public APIs?